### ISSUE TITLE
`[role=search]` test reset has insufficient specificity 

### ISSUE THREAD

**Issue by ffoodd** [ffoodd/a11y.css#274]
Example : [Julie Moynat's website](https://www.lelutinduweb.fr/).

Pretty easy to fix.

**Comment by ffoodd** [comment]
In fact it would only appear for input (as they're void tags and `@a11y-reset` cannot take that into account).
